README.md

```diff
@@ -134,7 +134,7 @@ Callback is called with `(err, data)`
 
 #### `feed.getBatch(start, end, [options], callback)`
 
-Get a range of blocks efficiently. Options include
+Get a range of blocks efficiently. End index is non-inclusive. Options include
```

Comment:
Another word for "non-inclusive" could be "exclusive"

Reply:
It's strange but somehow I feel a novice would understand `non-inclusive` more easily. But As you say.

Reply:
@perguth I don't have any power here, in math/CS I feel like intervals are usually called inclusive and exclusive

Reply:
Ok, I would stick to non-inclusive. It's also used a couple of lines down in this paragraph: https://github.com/mafintosh/hypercore/blob/5a46735bb95e96265dea32f2919a94da120b00fa/README.md#feeddownloadrange-callback